src/node_constants.cc

```diff
@@ -817,6 +817,12 @@ void DefineCryptoConstants(Local<Object> target) {
     NODE_DEFINE_CONSTANT(target, OPENSSL_VERSION_NUMBER);
 #endif
 
+#ifdef OPENSSL_IS_BORINGSSL
+#undef OPENSSL_IS_BORINGSSL
+#define OPENSSL_IS_BORINGSSL 1
+    NODE_DEFINE_CONSTANT(target, OPENSSL_IS_BORINGSSL);
+#endif
```

Comment:
I'm not a big fan of redefining the existing constant this way. Perhaps this could just be `crypto.constants.BORINGSSL` and the definition here could be:

```cpp
#ifdef OPENSSL_IS_BORINGSSL
  constexpr auto BORINGSSL = 1;
#else
  constexpr auto BORINGSSL = 0;
#endif
  NODE_DEFINE_CONSTANT(target, BORINGSSL);
```

Reply:
Alternatively, I do wonder if this is better exposed via `process.config` or `process.features`. We do have the existing `process.config.openssl_is_fips` and `process.config.openssl_quic` flags in `process.config`.